rest_framework_json_api/utils.py

```diff
@@ -236,15 +243,11 @@ def get_related_resource_type(relation):
                     relation_model = parent_model_relation.related.related_model
                 else:
                     relation_model = parent_model_relation.related.model
-            elif parent_model_relation_type is ManyToManyDescriptor:
```

Comment:
Dunno, I suppose it's a merging issue from [0ddf5ca](https://github.com/django-json-api/django-rest-framework-json-api/pull/211/commits/0ddf5ca15867d017cce46bd1fa65990820b9b2de#diff-a16d805758b2ef29db6bd4ff77e9ccfb). Any clue @astronouth7303?